packages/react/src/components/layout/Popin/Popin.stories.tsx

```diff
@@ -1,26 +1,32 @@
 import React from "react";
 import { useArgs } from "@storybook/client-api";
+import { action } from "@storybook/addon-actions";
 
 import Popin, { PopinProps } from "./index";
 import Text from "../../asorted/Text";
+import Button from "../../cta/Button";
 import theme from "../../../styles/theme";
 
 export default {
-  title: "Layout/Drawer/Popin",
+  title: "Layout/Popin",
   component: Popin,
-  argTypes: {
-    children: {
-      type: "text",
-      description: "Any valid React node",
-      required: true,
-      control: {
-        type: "text",
+  parameters: {
+    docs: {
+      description: {
```

Comment:
Some typos and suggestions:

`customisable` => `customizable`
`process<br />` missing a dot `process.<br/>`
`by calling the [...] components` => IMO calling a `method` would be more correct